content/cpp/concepts/math-functions/terms/tan/tan.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: 'tan()'
+Description: 'Returns the tangent of the argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`tan()`** function returns the tangent of the argument.
+
+## Syntax
+
+```cpp
+tan(x)
+```
+
+If the parameter `x` is equal to pi +- n*pi (Where n is a natural number), then a domain error will occur.
+
+## Example
+
+The following example uses the `tan()` function to find the tangent of `pi/4`:
```

Comment:
```suggestion
The following example uses the `tan()` function to find the tangent of `pi/4`:
```
```suggestion
The following example uses the `tan()` function to find the tangent of pi/4:
```

Reply:
I'd remove the backticks because "pi/4" isn't written how it appears in the code.  You might also mention that `M_PI` is a constant defined in the `cmath` header file.